portfolio/src/main/webapp/index.html

```diff
@@ -53,6 +53,18 @@ <h2>Hidden Talent</h2>
             </a>
         </div>
 
+        <div>
+            <form action="/data" method="POST">
```

Comment:
Not sure if the project expects the path to be "/data", but if you're posting a comment, it might be better to name this something like /comment.